tests/utils/test_io.py

```diff
@@ -49,6 +50,24 @@ def test_load_model_checkpoint(path):
     model = load_model(path)
     assert type(model) is SoapBpnn
 
+
+@pytest.mark.parametrize(
+    "path",
+    [
+        RESOURCES_PATH / "model-64-bit.ckpt",
+    ],
+)
+def test_load_model_checkpoint_wrong_version(path):
+    file = RESOURCES_PATH / "model-64-bit-version5000000.ckpt"
```

Comment:
please don't write in the current directory but move around to a tmpdir first

Reply:
but all the other test do that

Reply:
i mean the test for the command-line write in the same folder, from which this reads the checkpoint that transforms

Reply:
no, most tests start by changing to a temporary directory: https://github.com/metatensor/metatrain/blob/976a66d34082d2d65bf18a629232568dbb61b63a/tests/cli/test_eval_model.py#L31